test/fixtures/HostAPITests.jl

```diff
@@ -0,0 +1,152 @@
+module HostAPITests
```

Comment:
Same for this.

Reply:
This will be addressed in a later update and is now tracked in #191.